doc/usecases/plugins/spec_globbing.md

```diff
@@ -0,0 +1,61 @@
+# Use Case: Specification using Globbing
+
+## Summary
+
+- **Scope:** Configuration
+- **Level:** Developer Goal
+- **Actors:** Application developer
+- **Brief:** Writing generic specification using elektra globbing in key names.
+
+## Scenarios
+
+- **Precondition:** kdb is installed.
+- **Main success scenario:**
+  - Globbing is used in order to make specification generic.
```

Comment:
Per actual step only one point. Explanations of terms should be in glossary.

Reply:
Where I can find that glossary? Or do you mean a separate section in this document?

Reply:
It's `/doc/help/elektra-glossary.md` in the repo